parmed/tools/simulations/openmm.py

```diff
@@ -16,7 +16,10 @@
 import sys
 import warnings
 try:
-    from simtk.openmm.vec3 import Vec3
+    try:
+        from simtk.openmm.vec3 import Vec3
+    except ImportError:
+        from openmm.vec3 import Vec3
```

Comment:
I suggest reversing this, and likewise in similar imports.  First try importing without `simtk`, then import with it if that fails.  That way you won't get a deprecation warning when using 7.6.

Reply:
Ha, I actually just dropped support for OpenMM <7.6 altogether.

I'll hopefully finish up the CHARMM work soon so I can actually release ParmEd 4.0.